Textify/HtmlTraversal.cs

```diff
@@ -180,16 +193,41 @@ private void HandleElement(IElement element)
 
                     // Separate rows with an empty line
                     Write("\n\n");
-                    
+
                     break;
 
                 case "TD":
                 case "TH":
                     TraverseChildren(element);
-                    
+
                     // Separate table columns with a symbol
                     Write(" | ");
-                    
+
+                    break;
+
+                case "A":
+                    string hrefAttribute = element.GetAttribute("href");
+                    var linkIndex = -1;
+
+                    if (!string.IsNullOrWhiteSpace((hrefAttribute)))
+                    {
+                        if (links.Contains(hrefAttribute))
+                        {
+                            linkIndex = links.IndexOf(hrefAttribute) + 1;
+                        }
+                        else
+                        {
+                            links.Add(hrefAttribute);
+                            linkIndex = links.Count();
+                        }
+                    }
+
+                    TraverseChildren(element);
+
+                    if (linkIndex >= 0)
+                    {
+                        Write($" ({linkIndex})");
```

Comment:
Wouldn't it be better to have the index in square brackets, like `[1]`? I'm used to see brackets instead of parentheses, but if you a particular reason for your choice I'm open to that 😊

Reply:
I'm neutral to both. Generally I would have said round brackets in the text look nicer, but then again we use square at the bottom, so might as well use them in both places.